content/docs/1.6.0/deploy/install/install-with-helm-controller.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Install with Helm Controller
+weight: 10
+---
+
+In this section, you will learn how to install Longhorn with the HelmChart controller built into RKE2 and K3S.
```

Comment:
```suggestion
In this section, you will learn how to install Longhorn with the HelmChart controller built into RKE2 and K3s.
```

Reply:
Noted, for future reference.  For what it is worth, even https://longhorn.io/docs/1.6.0/advanced-resources/os-distro-specific/csi-on-k3s/ uses several variations: K3s, K3S, k3s.

Reply:
Done.